lib/player.js

```diff
@@ -1323,7 +1323,6 @@ shaka.Player = class extends shaka.util.FakeEventTarget {
     if (this.adManager_) {
       // The ad manager is specific to the video, so dispose of it.
```

Comment:
If we're no longer completely getting rid of the ad manager, this comment should be updated.
`// The ad manager is specific to the video, so detach it too.`
Something like that.

Reply:
Done!